pkgs/top-level/python-packages.nix

```diff
@@ -6380,6 +6380,8 @@ in {
 
   prometheus-flask-exporter = callPackage ../development/python-modules/prometheus-flask-exporter { };
 
+  prometheus-pve-exporter = callPackage ../development/python-modules/prometheus-pve-exporter { };
```

Comment:
If we don't add the package to pythonPackages we have more freedom in patching. Also I don't think it will be a dependency of something in the future.